controllers/clusterresources/awsendpointserviceprincipal_controller.go

```diff
@@ -34,7 +33,9 @@ import (
 	clusterresourcesv1beta1 "github.com/instaclustr/operator/apis/clusterresources/v1beta1"
 	"github.com/instaclustr/operator/pkg/instaclustr"
 	"github.com/instaclustr/operator/pkg/models"
+	"github.com/instaclustr/operator/pkg/ratelimiter"
 	"github.com/instaclustr/operator/pkg/scheduler"
```

Comment:
You have unsorted imports in the PR, please sort one

Reply:
Fixed